client/src/components/Upload/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { createRef } from 'react'
+
+export default function Modal() {
+  const toggled = createRef<HTMLDivElement>()
+
+  const handleToggleBody = () => {
+    console.log('ff')
```

Comment:
so we will just leave it with a console.log! if you want to do anything later then please leave a comment that explains so `// TODO: we still need to do so and so`